ttnn/cpp/ttnn/operations/data_movement/common/kernels/debug.hpp

```diff
@@ -0,0 +1,42 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+// This file contains common kernel functions used for debugging
+#pragma once
+#include "debug/dprint.h"
+namespace tt::data_movement::common {
+inline void print_bf16_pages(uint32_t l1_addr, uint32_t elts_per_page, uint32_t npages, uint32_t start = 0) {
```

Comment:
@mywoodstock i think you wanted to add something like this. Maybe we can add it to the correct place. @jaykru-tt can we move this to a common directory for all ops

Reply:
Sure, which directory?

Reply:
ttnn/operations/common or something like that?

Reply:
@jaykru-tt that sounds good to me. @mywoodstock do you agree?

Reply:
I'm going to keep this in place and relocate it in a future PR.